### PR TITLE
chore(metadata): script to update all metadata

### PR DESCRIPTION
**Pull request by pmast**

### CODE REVIEW

File: scripts/update-metadata.sh [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+layersConfig=$(cat $layersConfigFile)
+#keys=$(jq -r 'keys[]' <<< "$layersConfig" | head -n 1)
+
+# take optional layer ID from command line argument
+layerID=$1 
+
+# check if layer ID is set
+if [ -z $layerID ]
+then
+    keys=$(jq -r 'keys[]' <<< "$layersConfig")
```
Comment (pwambach) [inline scripts/update-metadata.sh:19]: can't we just say `keys=$(jq -r 'keys[]' $layersConfigFile)` here?

File: scripts/update-metadata.sh [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    echo $datasetId 
+
+    # download metadata file
+    path=gs://esa-cfs-tiles/$version/$datasetId/metadata.json
+    metadataFileName=$datasetId-metadata.json
+    gsutil cp $path $workingDir/$metadataFileName
+
+    # iterate over config for dataset
+    props=$(jq -r ".\"$datasetId\" | keys[]"  <<< "$layersConfig")
+    for prop in ${props}
```
Comment (pwambach) [inline scripts/update-metadata.sh:36]: seems to be possible to merge 2 objects with jq https://stackoverflow.com/a/24904276
but i'm also fine with this solution

File: scripts/update-metadata.sh (hunk 1 above)
Comment (pmast) [reply scripts/update-metadata.sh:19]: Yes, we could do this. But I am caching the content to save i/o operations probably overcautious?!

File: scripts/update-metadata.sh (hunk 2 above)
Comment (pmast) [reply scripts/update-metadata.sh:36]: Well found!

I will give the merge a try tomorrow, would save us the nested loop and fiddling around with tmp files.

File: scripts/update-metadata.sh (hunk 1 above)
Comment (pmast) [reply scripts/update-metadata.sh]: Using the file directly for readability

File: scripts/update-metadata.sh (hunk 2 above)
Comment (pmast) [reply scripts/update-metadata.sh]: Done!

**Pull request merged**